src/app/modules/dashboard/dashboard-nav-bar/dashboard-nav-bar.component.ts

```diff
@@ -12,12 +15,15 @@ export class DashboardNavBarComponent implements OnInit {
   @Input() items: DashboardNavBarData[] = [];
   @Input() show = false;
 
-  constructor() { }
+  constructor(private store: Store<AppState>) { }
 
   ngOnInit() { }
 
   redirect(redirectCallback: () => Promise<boolean>) {
     redirectCallback();
   }
 
+  signOut(): void {
+    this.store.dispatch(fromAuthorization.signOut());
+  }
```

Comment:
Czemu czasem przekazujesz tak jak wyże a czasem tak -> "this.store.dispatch(ErrorResponseActions.unauthorized);"

Reply:
Bardzo słuszna uwaga. ErrorResponseActions.unauthorized zwraca ActionCreator, więc powinien być wywołwany. Tutaj przez pomyłkę dispatchowałem funkcję zamiast obiektu akcji

Reply:
To będziesz to jeszcze poprawiał?

Reply:
Już poprawiłem